.Package.test.swift

```diff
@@ -2,6 +2,10 @@ import PackageDescription
 
 let package = Package(
     name: "ReactiveSwift",
+    targets: [
+        Target(name: "ReactiveSwift", dependencies: ["OSLocking"]),
+        Target(name: "OSLocking")
+    ],
```

Comment:
Could try to move the rest of the stuff. Though as a side note, these C stuff are introduced to dodge [compiler enforced `@available` runtime checks in `lock()` and `unlock()`](https://bugs.swift.org/browse/SR-4079).